Add a note to the documentation about pre-built HLLSketches (#13088)
apache/druid: docs/development/extensions-core/datasketches-hll.md

 }
```

> The `HLLSketchBuild` aggregator builds a datasketch from the input column specified. If used during ingestion, this
> will result in Druid storing pre-generated HLL Sketch objects in the datasource, rather than the original value itself.
> If used at query time on an existing dimension, the resulting column can be used as an intermediate dimension by the
> post-aggregators below.
> 

Reviewer: ```suggestion
The `HLLSketchBuild` aggregator builds a datasketch from the specified input column. When used during ingestion, Druid stores pre-generated HLL Sketch objects in the datasource instead of the original values.
When applied at query time on an existing dimension, you can use the resulting column as an intermediate dimension by the [post-aggregators](#post-aggregators).

```
Thanks for the clarification/ contribution @cloventt ! I've suggest some stylistic changes.

Reviewer 2: I made a few minor edits to the suggestion above for consistency of language and style. One question: What does "original values" refer to? The raw data that made up the sketch?

Reviewer: That was the way I understood it. It stores the sketch/aggregation.

Author: Style changes commited in df7d508.

 }
```

> The `HLLSketchMerge` aggregator can be used to ingest pre-generated sketches from an input dataset. For example, an
> earlier batch processing job can be used to generate the sketches before the data is sent to Druid. To support this
> behaviour, the sketches in the input dataset must be serialised to base64-encoded bytes. Then, in the native ingestion
> `MetricsSpec` the `HLLSketchMerge` must be specified for the input column as shown above.
> 

Reviewer: ```suggestion
You can use the `HLLSketchMerge` aggregator to ingest pre-generated sketches from an input dataset. For example, you can set up a batch processing job to generate the sketches before sending the data to Druid. You must serialize the sketches in the input dataset to base-64 encoded bytes. Then, specify `HLLSketchMerge` for the input column in the native ingestion`MetricsSpec`.

```
Stylistic suggestions. Also wonder if it might be helpful to have an example of the `MetricsSpec`.

Author: I'm not sure on the usefulness of an example - the code blob to be added to the `MetricsSpec` is already included directly above this paragraph.
